src/python_routing_v02/setup.py

```diff
@@ -0,0 +1,42 @@
+from distutils.core import setup
+from distutils.extension import Extension
+import sys
+import numpy as np
+
+"""
+If source ships with the cython generated .c code, then cython isn't a hard requirement
+This setup can use if if passed the --use-cython flag, otherwise it looks for the c source
```

Comment:
I wonder if a more robust logic would be to check for cython first, then, if cython isn't callable, check for the c files. That way, we can issue a warning if the version of cython is too old. I think we require cython >= 0.29.1, but recommend the latest version. This also means that we use freshly generated c files.

Reply:
That is actually part of the problem, once the C code is generated, the cython version doesn't matter, the extension is built based on the static C code.  If you re-cythonize each time you distribute, then different versions of cython may produce different C code which may or may not work.

The way it exists now gives someone the ability to use cython and rebuild if they know what they are doing, but for all other cases, it just builds the extension from the existing C code.

I am going to remove the .c files from this PR FOR NOW until the development gets a little more complete.